bip-0067.mediawiki

```diff
@@ -0,0 +1,123 @@
+** 027735a29bae7780a9755fae7a1c4374c656ac6a69ea9f3697fda61bb99a4f3e77
+** 02e2cc6bd5f45edd43bebe7cb9b675f0ce9ed3efe613b177588290ad188d11b404
+* Sorted: 
+** 02632b12f4ac5b1d1b72b2a3b508c19172de44f6f46bcee50ba33f3f9291e47ed0
+** 027735a29bae7780a9755fae7a1c4374c656ac6a69ea9f3697fda61bb99a4f3e77
+** 02e2cc6bd5f45edd43bebe7cb9b675f0ce9ed3efe613b177588290ad188d11b404
+* Script
+** 522102632b12f4ac5b1d1b72b2a3b508c19172de44f6f46bcee50ba33f3f9291e47ed021027735a29bae7780a9755fae7a1c4374c656ac6a69ea9f3697fda61bb99a4f3e772102e2cc6bd5f45edd43bebe7cb9b675f0ce9ed3efe613b177588290ad188d11b40453ae
+* Address
+** 3CKHTjBKxCARLzwABMu9yD85kvtm7WnMfH
+
+Vector 3:
+* List:    
+** 030000000000000000000000000000000000004141414141414141414141414141
+** 020000000000000000000000000000000000004141414141414141414141414141
+** 020000000000000000000000000000000000004141414141414141414141414140
+** 030000000000000000000000000000000000004141414141414141414141414140
+* Sorted: 
+** 020000000000000000000000000000000000004141414141414141414141414140
+** 020000000000000000000000000000000000004141414141414141414141414141
+** 030000000000000000000000000000000000004141414141414141414141414140
+** 030000000000000000000000000000000000004141414141414141414141414141
+* Script
+** 522102000000000000000000000000000000000000414141414141414141414141414021020000000000000000000000000000000000004141414141414141414141414141210300000000000000000000000000000000000041414141414141414141414141402103000000000000000000000000000000000000414141414141414141414141414154ae
+* Address
+** 32V85igBri9zcfBRVupVvwK18NFtS37FuD
+
+Vector 4: (from bitcore)
+* List: 
+** 022df8750480ad5b26950b25c7ba79d3e37d75f640f8e5d9bcd5b150a0f85014da
+** 03e3818b65bcc73a7d64064106a859cc1a5a728c4345ff0b641209fba0d90de6e9 
+** 021f2f6e1e50cb6a953935c3601284925decd3fd21bc445712576873fb8c6ebc18
+* Sorted: 
+** 021f2f6e1e50cb6a953935c3601284925decd3fd21bc445712576873fb8c6ebc18
+** 022df8750480ad5b26950b25c7ba79d3e37d75f640f8e5d9bcd5b150a0f85014da
+** 03e3818b65bcc73a7d64064106a859cc1a5a728c4345ff0b641209fba0d90de6e9
+* Script
+** 5221021f2f6e1e50cb6a953935c3601284925decd3fd21bc445712576873fb8c6ebc1821022df8750480ad5b26950b25c7ba79d3e37d75f640f8e5d9bcd5b150a0f85014da2103e3818b65bcc73a7d64064106a859cc1a5a728c4345ff0b641209fba0d90de6e953ae
+* Address
+** 3Q4sF6tv9wsdqu2NtARzNCpQgwifm2rAba
+
+==Usage & Implementations==
+* BIP45 - Structure for Deterministic P2SH Multisignature Wallets - https://github.com/bitcoin/bips/blob/master/bip-0045.mediawiki#address-generation-procedure
+* Bitcore - https://github.com/bitpay/bitcore/blob/50a868cb8cdf2be04bb1c5bf4bcc064cc06f5888/lib/script/script.js#L541
+* Haskoin - https://github.com/haskoin/haskoin/blob/master/Network/Haskoin/Script/Parser.hs#L112-122
+* Armory - https://github.com/etotheipi/BitcoinArmory/blob/268db0f3fa20c989057bd43343a43b2edbe89aeb/armoryengine/ArmoryUtils.py#L1441
+* Multisignature Brainwallet - http://ms-brainwallet.org/
```

Comment:
the reason for listing as many as possible is initially too show there's already a bunch of wallets and libs following this. 

2nd is imo that this BIP aims to close the gap for compatibility, I think it's relevant. 

imo it should stay in until the BIP is accepted, since it's relevant for the discussion